circle.yml

```diff
@@ -1,28 +1,15 @@
 version: 2.1
 
-# usually we don't build Mac app - it takes a long time
-# but sometimes we want to really confirm we are doing the right thing
-# so just add your branch to the list here to build and test on Mac
-macBuildFilters: &macBuildFilters
-  filters:
-    branches:
-      only:
-        - develop
-
```

Comment:
this was handy to have at the top, instead of having to go to the bottom to build on a branch. since it's pretty common to want to turn on the full build process (`create-build-artifacts`, posting the cdn.cypress.io comment to GitHub commit, example repo tests) for 1 or all 3 OS's to test changes in a branch.

can you add a clear way to enable/disable the full builds?

Reply:
done: [`197ec25` (#19037)](https://github.com/cypress-io/cypress/pull/19037/commits/197ec254f25e3c97a2d7e6c3b3f52d7ab9eb81cf)